app/views/brexit_landing_page/_page_header.html.erb

```diff
@@ -46,6 +46,12 @@
         dark_icon: true,
         margin_bottom: 4,
         font_size: "m",
+        data: {
+          "module": "gem-track-click",
+          "track-category": "brexit-landing-page",
+          "track-action": t("brexit_landing_page.page_header_links.guidance_for_businesses.path"),
+          "track-label": t("brexit_landing_page.page_header_links.guidance_for_businesses.text")
+        }
```

Comment:
very minor comment, but lines 52 and 53 are a bit long. We could use a variable to store the track-action and track-label values like we do [here](https://github.com/alphagov/collections/blob/62fad20d8fad6571e323e791c433e065e7540c60/app/views/brexit_landing_page/_page_header.html.erb#L51-L52)